src/components/expandable-card/expandable-card.scss

```diff
@@ -0,0 +1,92 @@
+@use "@/styles/responsive-mixins";
+@use "@/styles/mixins";
+
+.expandable-card {
+  --_text-color: var(--text-color, var(--black));
+  --_action-color: var(--action-color, var(--primary));
+  --_content-color: var(--content-color, var(--white));
+  display: flex;
+  flex-direction: column;
+  border-radius: 8px;
+  overflow: hidden;
+  color: var(--_text-color);
+
+  &-content {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+    padding: 24px;
+    background-color: var(--_content-color);
+  }
+
+  &-title {
+    flex-shrink: 0;
+    @include mixins.h5;
+  }
+
+  &-body {
+    flex-grow: 1;
+  }
+
+  &-action {
+    @include mixins.h5;
+    font-weight: 600;
+    flex-shrink: 0;
+    margin-top: 16px;
+    text-decoration: none;
+    color: var(--_action-color);
+
+    &:hover,
+    &:focus {
+      text-decoration: underline;
+    }
+  }
+
+  &:hover &-image {
+    transform: scale(1.25);
+  }
+
+  .expandable-button-link.outline {
```

Comment:
Why is this .outline class needed here? Let's just use button tag, not Button component.